neuro_extras/main.py

```diff
@@ -93,7 +93,19 @@ def _create_builder_container(
         image_ref: str,
         build_args: Sequence[str] = (),
     ) -> neuro_api.Container:
-        command = f"--dockerfile={dockerfile_path} --destination={image_ref}"
+
+        async with neuro_api.get() as client:
+            registry_url_no_https = str(client.config.registry_url).replace(
+                "https://", ""
+            )
+            cache_repo = (
+                f"{registry_url_no_https}/{client.config.username}/layer-cache/cache"
+            )
```

Comment:
I guess we could reuse `parse_image_ref` here.

Reply:
We could, but we still have to remove the tag from its output. I've made the change and pushed, let me know if it looks better to you.